python/ambassador/ir/irhost.py

```diff
@@ -72,6 +73,16 @@ def setup(self, ir: 'IR', aconf: Config) -> bool:
         self.insecure_action = insecure_policy.get('action', 'Redirect')
         self.insecure_addl_port: Optional[int] = insecure_policy.get('additionalPort', None)
 
+        # If we have no mappingSelector, check for selector.
+        mapsel = self.get('mappingSelector', None)
+
+        if not mapsel:
+            mapsel = self.get('selector', None)
+
+            if mapsel:
```

Comment:
Stupid curiosity: You and I have very different styles for where to put blank lines. So, to pick on this as a typical example (and not that I think I could articulate my style): Why's a blank line belong here?

Reply:
Uh. 🤔 

... 🤔  ...

...I think it comes down to where I think the breaks in thoughts are. The whitespace is there to make it simpler to read later: groups of lines with no blank lines are things I think are OK to chunk together and either consider or skip as a single unit, but the blank lines mark "oh no, this is a new concept – here's a place where you need to pay more attention even if you were writing the previous chunk off as not relevant for your current study."

I think I should think a bit more about how to articulate that...